src/praisonai/praisonai/bots/_session.py

```
@@ -1268,6 +1268,74 @@ def get_user_ids(self) -> List[str]:
         return list(self._histories.keys())
 
 
+def resolve_durable_store_dir(platform: str = "") -> "Path":
```

Ruff (0.15.18)

[error] Undefined name `Path`

(F821)